packages/modular-scripts/src/config/jest/index.ts

```diff
@@ -195,5 +196,16 @@ export function createJestConfig(
       moduleNameMapper: mergedMapper,
     });
   }
+
+  // don't typecheck tests in CI
+  if (isCi) {
+    jestConfig.globals = {
+      'ts-jest': {
+        diagnostics: false,
+        isolatedModules: true,
```

Comment:
I think we always want this to be true? Because it matches the `esbuild-loader` functionality. I thought this was already the case from the `tsconfig.json` we use for `modular-scripts`, but I guess it would be good to enforce this naturally.

Reply:
ts-jest has this defaulted to false https://kulshekhar.github.io/ts-jest/docs/getting-started/options/isolatedModules

This is what enables and disables typechecking with ts-jest. I thought we wanted to keep typechecking for local tests?

Reply:
Oh I guess this is different to `tsconfig.compilerOptions.isolatedModules` then?

Reply:
Yeah I think this is a ts-jest option that needs to be turned off explicitly